Signed-off-by: Ondřej Vašík <ovasik@redhat.com>
Signed-off-by: Jim Meyering <meyering@redhat.com>

* src/echo.c (AUTHORS) : Use bash builtin echo authors instead of FIXME unknown
* src/basename.c (AUTHORS): List David as the author.
* AUTHORS: Update here, too.
---
 AUTHORS        |    4 ++--
 src/basename.c |    2 +-
 src/echo.c     |    4 +++-
 3 files changed, 6 insertions(+), 4 deletions(-)
diff --git a/src/basename.c b/src/basename.c
index 38e8879..69b708f 100644
--- a/src/basename.c
+++ b/src/basename.c
@@ -37,7 +37,7 @@
 /* The official name of this program (e.g., no `g' prefix).  */
 #define PROGRAM_NAME "basename"
 
-#define AUTHORS proper_name ("FIXME unknown")
+#define AUTHORS proper_name ("David MacKenzie")
 
 /* The name this program was run with. */
 char *program_name;
diff --git a/AUTHORS b/AUTHORS
index 404cf70..666edc1 100644
--- a/AUTHORS
+++ b/AUTHORS
@@ -3,7 +3,7 @@ each followed by the name(s) of its author(s).
 
 arch: David MacKenzie, Karel Zak
 base64: Simon Josefsson
-basename: FIXME unknown
+basename: David MacKenzie
 cat: Torbjörn Granlund, Richard M. Stallman
 chcon: Russell Coker, Jim Meyering
 chgrp: David MacKenzie, Jim Meyering
@@ -22,7 +22,7 @@ dir: Richard M. Stallman, David MacKenzie
 dircolors: H. Peter Anvin
 dirname: David MacKenzie, Jim Meyering
 du: Torbjörn Granlund, David MacKenzie, Paul Eggert, Jim Meyering
-echo: FIXME unknown
+echo: Brian Fox, Chet Ramey
 env: Richard Mlynarik, David MacKenzie
 expand: David MacKenzie
 expr: Mike Parker
diff --git a/src/echo.c b/src/echo.c
index ebbf5b8..11e648e 100644
--- a/src/echo.c
+++ b/src/echo.c
@@ -24,7 +24,9 @@
 /* The official name of this program (e.g., no `g' prefix).  */
 #define PROGRAM_NAME "echo"
 
-#define AUTHORS proper_name ("FIXME unknown")
+#define AUTHORS \
+  proper_name ("Brian Fox"), \
+  proper_name ("Chet Ramey")
 
 /* echo [-neE] [arg ...]
 Output the ARGs.  If -n is specified, the trailing newline is
